src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -365,12 +355,32 @@ export default {
         openNotice() {
             this.notice = true;
         },
-
-        toggleOption(key) {
-            if (this.ballot.max_options === 1) {
-                this.votes = this.votes.includes(key) ? [key] : [];
+        shouldDisableCheckbox(key) {
+            return (
+                !this.isAuthenticated ||
+                !this.isBallotOpened(this.ballot) ||
+                (this.votes.length === this.ballot.max_options && !this.votes.includes(key))
+            );
+        },
+        displayBallotSelectionText() {
+            if (this.votes.length === 0) {
+                if (this.ballot.max_options > 1) {
+                    return `You can select from ${this.ballot.min_options} to ${this.ballot.max_options} options`;
+                }
+                return 'You can select 1 option only';
             }
-            this.userCanVote = this.votes.length > 0;
+            if (this.votes.length === this.ballot.max_options) {
+                return "You've selected the maximum allowed options";
+            }
+            return `You can select ${this.ballot.max_options - this.votes.length} more options`;
```

Comment:
```suggestion
            return this.$t('pages.trails.ballots.youCanSelectMoreOps', {
               quantity: (this.ballot.max_options - this.votes.length).toString()
            });
```

then you must include the following entry on index.js 

```
     youCanSelectMoreOps: 'You can select {quantity} more options',
```